### PR TITLE
Fix processor when scope provider in another package

### PR DESCRIPTION
**Pull request by takahirom**
Thanks for the great library!

If the scope provider is in another package, it couldn't be imported and the build failed.
This was fixed by using KotlinPoet's Members format.
https://square.github.io/kotlinpoet/#m-for-members

### CODE REVIEW

File: koru-processor/src/jvmMain/kotlin/com/futuremind/koru/processor/WrapperClassBuilder.kt [hunk 1]
```diff
@@ -74,15 +74,35 @@ class WrapperClassBuilder(
...
-            "return %T(${scopeProviderSpec?.name}) { ${originalFunSpec.asInvocation()} }",
-            SuspendWrapper::class
-        )
-        originalFunSpec.returnType.isFlow -> this.addStatement(
-            "return %T(${scopeProviderSpec?.name}, ${originalFunSpec.asInvocation()})",
-            FlowWrapper::class
-        )
+    private fun FunSpec.Builder.addFunctionBody(originalFunSpec: FunSpec): FunSpec.Builder = when {
+        this.isSuspend -> {
+            if (scopeProviderMemberName != null) {
```
Comment (micHar) [inline koru-processor/src/jvmMain/kotlin/com/futuremind/koru/processor/WrapperClassBuilder.kt]: I wish this conditional statement could be simplified somehow, but `addStatement` doesn't accept nullable arguments.

On the other hand, maybe the `scopeProviderMemberName: MemberName` should just never be nullable, but just **contain** `null` if no ScopeProvider is used. Not sure how it's gonna work with kotlinpoet yet.

**Review by micHar — COMMENTED** [review]
Good catch, thank you for the PR!

I have one slight doubt that I expressed in a file comment. I you have time to look if it can be done, please update the PR. If not, I will probably try that tomorrow, merge and release new version (busy today).

File: koru-processor/src/jvmMain/kotlin/com/futuremind/koru/processor/WrapperClassBuilder.kt (hunk 1 above)
Comment (takahirom) [reply koru-processor/src/jvmMain/kotlin/com/futuremind/koru/processor/WrapperClassBuilder.kt]: The first commit simply made scopeProviderMemberName: MemberName non-null, but that didn't work. I'll think of a way to make it a little simpler. 👀

**Comment by takahirom** [comment]
I tried using CodeBlock. How is it?

**Comment by micHar** [comment]
@takahirom thank you very much for this contribution. I refactored it a bit and merged. Will hopefully release 0.3.5 today.

**Comment by takahirom** [comment]
Thank you for the early release!

**Pull request merged**